.travis.yml

```diff
@@ -116,7 +116,16 @@ before_install:
       sudo apt-get update;
       sudo apt-get install sagemath-upstream-binary;
     fi
+#The install cycle below is to test installation on systems without setuptools.
 install:
+  - if [[ "${TEST_AUTOWRAP}" == "true" ]]; then
+      pip uninstall -y setuptools;
+      sudo apt-get remove --purge python-setuptool;
```

Comment:
`E: Unable to locate package python-setuptool`

Should there be an `s` on the end?